script/build-artifact.sh

```diff
@@ -44,9 +44,15 @@ mkdir -p "${STAGING}" "${EXTRAS_STAGING}"
 
 echo "Creating \"${TARBALL}\" and \"${EXTRAS_TARBALL}\""
 
+# Use linux config file as default
+RELEASE_FOLDER="linux"
```

Comment:
I think this script currently works with darwin.... maybe this release folder can be named posix or something? Or do we duplicate it for darwin?

Reply:
I'll update to posix